homeassistant/components/envertech_evt800/sensor.py

```diff
@@ -0,0 +1,257 @@
+"""Envertech EVT800 sensor."""
+
+from typing import Any
+
+import pyenvertechevt800
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    UnitOfElectricCurrent,
+    UnitOfElectricPotential,
+    UnitOfEnergy,
+    UnitOfFrequency,
+    UnitOfPower,
+    UnitOfTemperature,
+)
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from .const import (
+    DOMAIN,
+    ENVERTECH_EVT800_COORDINATOR,
+    ENVERTECH_EVT800_DEVICE_INFO,
+    ENVERTECH_EVT800_OBJECT,
+)
+
+SENSORS: dict[str, SensorEntityDescription] = {
+    "timestamp": SensorEntityDescription(
+        key="timestamp",
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=0,
+        entity_registry_enabled_default=False,
+        entity_registry_visible_default=False,
+        translation_key="timestamp",
+    ),
+    "id_1": SensorEntityDescription(
+        key="id_1",
+        entity_registry_enabled_default=False,
+        entity_registry_visible_default=True,
+        translation_key="mptt_id_1",
+    ),
+    "id_2": SensorEntityDescription(
+        key="id_2",
+        entity_registry_enabled_default=False,
+        entity_registry_visible_default=True,
+        translation_key="mptt_id_2",
+    ),
+    "sw-version": SensorEntityDescription(
+        key="sw-version",
+        entity_registry_enabled_default=False,
+        entity_registry_visible_default=True,
+        translation_key="software_version",
+    ),
```

Comment:
Why is this an entity? You could have put this in the device info instead of a hardcoded value